plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestDynamicTable.java

```diff
@@ -139,11 +139,12 @@ public void testDoubleWithScientificNotation()
     @Test
     public void testFilterWithCast()
     {
+        // UPDATE: Pinot 0.11.0 automatically casts constants during parsing
```

Comment:
can be removed?

Reply:
Maybe it's safer to keep it, to verify the behavior? Or did you mean just remove the comment?

Reply:
I meant the comment.